README.md

```diff
@@ -1,14 +1,6 @@
-# AI-in-a-Box Accelerator Name
-
-<!-- <div style="display: flex;">
```

Comment:
should we remove this from the aib template as well?

Reply:
The template has this as a placeholder, to replace in the new project.